nodeJS/express/views.md

```diff
@@ -147,48 +161,37 @@ You can insert this component into another EJS file like so:
 </html>
 ```
 
-This can be used to include headers and footers in all of your pages, for example.
-
-Note that the navbar expects a `links` value. To pass this data into the navbar, you can pass it when rendering `index.ejs` which contains the navbar. Modify `app.js` such that a `links` object is defined and passed into the `render` function in the `"/"` route handler:
-
-```javascript
-// app.js
-const links = [
-  { href: "/", text: "Home" },
-  { href: "about", text: "About" },
-];
+This will include the navbar with links in `index.ejs`. Reusable templates can be used to include headers and footers in all of your pages.
 
-app.get("/", (req, res) => {
-  res.render("index", { links: links });
-});
-```
+Note the use of the raw output tag `<%-` with the `include` which is used to avoid double-escaping the HTML output.
 
-Here's another example of how to use `includes` to dynamically render a list of variables:
+Let's use `includes` to dynamically render a list of variables a different way. Add `const users = ["Bob", "Bill", "Biff"];` to `app.js` just below the `links` variable. Then create a new view called `user.ejs` in the `views` directory and add the following `<li><%= user %></li>` line. Finally add the following block to `index.ejs`:
 
 ```ejs
 <ul>
   <% users.forEach((user) => { %>
-    <%- include('user/show', {user: user}); %>
+    <%- include('user', {user: user}); %>
   <% }); %>
 </ul>
 ```
 
+If successful, Bob, Bill and Biff will be visible when rendering `index.ejs`.
+
 <div class="lesson-note lesson-note--tip" markdown="1">
 
 #### Directories within the views folder
 
-We can have nested directories of EJS template files within the views. For example, to render the template file `./views/user/show.ejs`, we'll need to provide the relative path like so:
+Let's create nested directories of EJS template files within the views. Change the `user.ejs` in the `views` directory to `users/user.ejs`, and in `index.ejs` change the path from `user` to `users/user` in the `users.forEach` block:
 
 ```javascript
-// in res.render
-res.render("user/show");
-
 // in include
```

Comment:
Currently it has 
````md
```javascript
// in include
````
Which should now be changed the following, given the contents change:
````md
```ejs
<!-- index.ejs -->
````

Reply:
Thanks for the catch, I'll try to be more mindful of needing to change the language when changing a codeblock.